src/content/docs/new-relic-solutions/get-started/intro-new-relic2.mdx

```diff
@@ -0,0 +1,195 @@
+
+    Some of the [quickstarts](https://newrelic.com/instant-observability) include pre-configured alerts that save you time setting up monitoring tools.
+
+    You can also check [our tutorial](/docs/tutorial-create-alerts/create-new-relic-alerts/) on getting started with New Relic alerts.
+    
+  </Step>
+
+  <Step>
+
+    ## Add users
+
+    You can add the rest of your team to New Relic. Use the **User management** UI to [add users](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-ui-and-tasks#where). See [User type](/docs/accounts/accounts-billing/new-relic-one-user-management/user-type) to learn more about the different types of users. You can also check [our tutorial](/docs/accounts/accounts-billing/new-relic-one-user-management/account-user-mgmt-tutorial) on adding users and managing user access.
+  </Step>
+</Steps>
+
+
+
+# Capabilities [#capabities]
+
+Our platform offers a comprehensive suite of capabilities to help organizations monitor, troubleshoot, and optimize the performance of their digital environments. Here's an explanation of New Relic's key capabilities. If you want to learn more, explore our [tutorials](/docs/new-relic-solutions/tutorial-landing-page/) pages. 
+
+
+## Alerts & AI [#alerts]
+
+Alerts help you identify issues in your applications and code. They show you how these issues affect your users. When you receive notifications of your alerts, you can stop problems and prevent widespread user disruptions. Being aware of issues early can lead to better user experiences.
+
+Let us help you know what's critical, manage the suppressed noise, and mitigate alert fatigue.
+
+<img
+  title="New Relic alerts main page"
+  alt="New Relic alerts main page"
+  src={NRAlertsMain}
+/>
+
+
+
+## APM & Services [#apm]
+
+New Relic instruments your application at the code level through the use of one of our many language agents. These agents collect metrics from your application and send them to our application performance monitoring (APM) , allowing you to monitor it using pre-built dashboards.
+
+The <InlinePopover type="apm" /> capability provides real-time insights into the performance of applications, services, and microservices across various programming languages and frameworks.
+
+It offers detailed transaction traces, code-level visibility, and performance metrics to identify bottlenecks, optimize application performance, and enhance user experience.
+
+<img
+  title="New Relic APM & Services main page"
+  alt="New Relic APM & Services main page"
+  src={NRApmMain}
+/>
+
+
+## Browser [#browser]
+
```

Comment:
I'd drop the word "capability" throughout in favor of browser monitoring.

For example, "Are you concerned about your website's performance during peak traffic hours? Use our browser monitoring to pinpoint the specific React component or JavaScript function that's making your site fall over."